information_models/dp_capaciteitskaart.schema.linkml.yml

```diff
@@ -253,12 +253,41 @@ classes:
         required: false
       congestion_url:
         description: URL for additional (congestion) information for this
-          EnergyArea.
+          EnergyArea. Only use this attribute if only one URL is required.
+          Otherwise, add more congestion information using
+          this:EnergyArea.CongestionInfo.
         range: string
         slot_uri: this:EnergyArea.congestionURL
         required: false
         multivalued: false
+      congestion_info:
+        description: Additional (congestion) information for this EnergyArea.
+          Use this to add multiple documentation and/or descriptions.
+        range: CongestionInfo
+        slot_uri: this:EnergyArea.CongestionInfo
+        required: false
+        multivalued: true
     class_uri: cim:EnergyArea
+  CongestionInfo:
```

Comment:
Dit wordt onnodig complex en rommelig denk ik. Volstaat het niet gewoon om te zeggen: `congestion_info` met `range: string`, waar evt. een URL in kan staan of niet.

Evt. kun je nog kijken naar:

https://linkml.io/linkml-model/latest/docs/AltDescription/

Just a thought, anders gewoon doorrammen.